stl/inc/xcall_once.h

```diff
@@ -31,6 +31,8 @@ using _Execute_once_fp_t = int(__stdcall*)(void*, void*, void**);
 
 _CRTIMP2_PURE int __CLRCALL_PURE_OR_CDECL _Execute_once(
     once_flag& _Flag, _Execute_once_fp_t _Callback, void* _Pv) noexcept;
+int __CLRCALL_PURE_OR_CDECL __std_execute_once_begin(once_flag& _Once_flag, int& _Pending, bool& fallback) noexcept;
+int __CLRCALL_PURE_OR_CDECL __std_execute_once_complete(once_flag& _Once_flag, unsigned long _Flags) noexcept;

```

Comment:
These functions are within `namespace std`, where we conventionally name them `_Meow`. We use `__std_meow` for `extern "C"` functions in the global namespace.